.github/workflows/test_dev.yml

```diff
@@ -39,29 +43,34 @@ jobs:
           python -m pip install --upgrade pip
           pip install .[tests]
           pip install ipykernel
-              
-      - name: Install qvm
+      - name: Setup IBMQ account
+        env:
+          IBMQ_TOKEN: ${{ secrets.IBMQ_TOKEN }} 
         run: |
-          find /usr/lib -name "libffi.so*"
-          sudo ln -s /usr/lib/x86_64-linux-gnu/libffi.so /usr/lib/x86_64-linux-gnu/libffi.so.6
-          wget -q https://downloads.rigetti.com/qcs-sdk/forest-sdk-2.23.0-linux-deb.tar.bz2
-          tar -xf forest-sdk-2.23.0-linux-deb.tar.bz2
-          cd forest-sdk-2.23.0-linux-deb/
-          sudo apt-get update
-          sudo apt-get install -y screen libblas-dev libblas3 libgfortran5 liblapack-dev liblapack3
-          sudo ./forest-sdk-2.23.0-linux-deb.run
-          qvm --version
-          quilc --version
-          echo "Starting qvm and quilc"
-          screen -d -m qvm -S
-          screen -d -m quilc -S
-          cd ..
-          
+          IBMQ_TOKEN=$IBMQ_TOKEN
+          source env/bin/activate
+          python -c'from qiskit import IBMQ; import os; IBMQ.save_account(os.environ.get("IBMQ_TOKEN"))'
+      - name: Setup AWS
+        env:
+          ACCESS_KEY: ${{ secrets.ACCESS_KEY }} 
+          SECRET_ACCESS_KEY: ${{ secrets.SECRET_ACCESS_KEY }} 
+        run: |
+          python -m pip install awscli
+          aws configure set aws_access_key_id $ACCESS_KEY
+          aws configure set aws_secret_access_key $SECRET_ACCESS_KEY
+          aws configure set region us-west-1
+          aws configure set output json
+      - name: Build the docker stack
+        run: |
+          docker login -u AWS -p $(aws ecr get-login-password --region us-east-1) 292282985366.dkr.ecr.us-east-1.amazonaws.com
+          docker network create local_bridge
+          docker build -t amazon-braket-oq-dev .
+          docker-compose up -d
       - name: Run tests
         run: |
           source env/bin/activate
           ipython kernel install --name "env" --user
-          python -m pytest -v -m 'not (qpu or api or docker_aws)' --cov --cov-report=xml:coverage.xml
+          pytest tests/ -v -m '(not qpu or sim)' --cov --cov-report=xml:coverage.xml
```

Comment:
We need this line also in all the main-tests!

Reply:
update on this? @Q-lds, do you want me to implement this edit on all `test_xyz.yml`?

Reply:
I think we can do it in a separate PR. It won't be easy, if we want to have all of them also for MacOs and Win tests

Reply:
UPDATE: discussed separately and will be fixed later